### PR TITLE
DDI 408 [Contextual Doc] GTM Guide - Add EU

### PR DESCRIPTION
**Pull request by Mercy811**
# Amplitude Developer Docs PR

## Description

- Rephrased the description of `init` tag in GTM. 
- Added EU config instructions

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [x] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [x] My documentation follows the style guidelines of this project.
- [x] I previewed my documentation on a local server using `mkdocs serve`.
- [x] Running `mkdocs serve` didn't generate any failures.
- [x] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/sources/google-tag-manager-client.md [hunk 1]
```diff
@@ -8,8 +8,12 @@ This is the client-side Google Tag Manager Template for Amplitude Analytics. The
 !!!info Resources
     [:simple-googletagmanager: GTM Template Gallery](https://tagmanager.google.com/gallery/#/owners/amplitude/templates/amplitude-browser-sdk-gtm-template) · [:material-github: GitHub](https://github.com/amplitude/amplitude-browser-sdk-gtm-template)
 
-With this template, you can defer the `init` tag until you receive a signal, like a consent grant. It's also common to fire the `init` tag using `All Pages` or `Initialization - All Pages` triggers. But make sure to fire the `init` tag **before** any other tags. The legacy template automatically calls init, so there's no need to do it yourself.
+It’s important to create an `init` tag. It's common to fire the `init` tag using `All Pages` or `Initialization - All Pages` triggers. With this template, you can also defer the `init` tag until you receive a signal, such as a consent grant. But notice that all other tags wait for the `init` tag to fire before they can be sent to Amplitude. The legacy template automatically calls init, so there's no need to do it yourself.
```
Comment (yuhao900914) [inline docs/data/sources/google-tag-manager-client.md:11]: Different from Amplitude Analytics Legacy, in Amplitude Analytics Browser SDK, we separate out the amplitude init call. You must create a separate tag for the `init`.  It's common to fire the `init` tag using `All Pages` or `Initialization - All Pages` triggers.  But you can also fire the init tag based on your customized triggers, such as a consent grant or others to defer the `init` call. But notice that all other tags wait for the `init` tag to fire before they can be sent to Amplitude. The legacy template automatically calls init, so there's no need to do it yourself.

Comment (yuhao900914) [reply docs/data/sources/google-tag-manager-client.md:11]: Not good at words...  Leave this as a suggestion.

Comment (yuhao900914) [reply docs/data/sources/google-tag-manager-client.md:11]: Others look good to me.

Comment (Mercy811) [reply docs/data/sources/google-tag-manager-client.md:11]: Thanks for your comment! 

I would like to start with the call out to create an `init` tag first as it's the point of this whole paragraph and put legacy SDK at the end. 

Added "customized triggers" to align with the fact that users can actually create customized triggers.

**Review by yuhao900914 — APPROVED** [review]

**Pull request merged**